application.go

```diff
@@ -72,132 +72,136 @@ type Application struct {
 	LastTaskFailure       *LastTaskFailure    `json:"lastTaskFailure,omitempty"`
 }
 
+// ApplicationVersions ... a collection of application versions for a specific app in marathin
 type ApplicationVersions struct {
 	Versions []string `json:"versions"`
 }
 
+// ApplicationVersion ... the application version response from marathon
 type ApplicationVersion struct {
 	Version string `json:"version"`
 }
 
+// VersionInfo ... the application versioning details from marathon
 type VersionInfo struct {
 	LastScalingAt      string `json:"lastScalingAt,omitempty"`
 	LastConfigChangeAt string `json:"lastConfigChangeAt,omitempty"`
 }
 
+// NewDockerApplication ... create a default docker application
```

Comment:
I think the comments should be in the form `X does stuff`. When `golint` says that their should be in the form `X ...` it means that instead of `...` there should be some actual text.
For example: https://github.com/golang/glog/blob/master/glog.go#L164
In this case it could be: `NewDockerApplication creates a default docker application`.
It's a minor thing, however it might be a bit easier to read.